### PR TITLE
Fixes #8 Added Node Slack Bot

### LINKED ISSUE #8: Creating an AskSusi Slack bot
@mariobehling I'd like to create a Slack bot for AskSusi. I'll work on it on a different repo. Once it is made, we can just fork my repo, or I'll just work on a new repo asksusi_slack.

Once we have enough bots, we can integrate all of them in one single repository.

### PR DESCRIPTION
**Pull request by shivenmian**
@sudheesh001 @mariobehling Added slack bot. As of now it runs only when I start it up from my terminal. Will deploy to heroku later.

### CODE REVIEW

**Comment by sudheesh001** [comment]
> Hardcoding the trigger word as "Susi" made me a bit uncomfortable. 

Why is the hardcoding required ? Is this an issue with the API ? I don't find the same problem when I use the susi API and see the responses

**Comment by shivenmian** [comment]
@sudheesh001 The thing is, the API has to be called only when the message starts with "Susi". So when I set up the webhook, there was an option for "Trigger Words", i.e it should POST to the API when the message starts with the Trigger word, so I added "Susi" as trigger word.

Now that I think of it, I can just evaluate ```req.body.text``` and split it and check if the message starts with Susi or not. But it's still working with keeping the Trigger word setting.

Is this fine or do I add the ```req.body.text contains Susi``` check in the code?

**Comment by sudheesh001** [comment]
Why can't we use `mentions` like `@susi` for the same, can we use every mentioned message of `susi` to trigger rather than reading every message and discarding the ones we don't need ?

**Comment by mariobehling** [comment]
Yes, `@susi!` seems the more obvious choice.

File: slack/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+//test msg
+app.get('/', function (req, res) {
+	res.send('Susi says Hello!');
+});
+
+//startup
+app.listen(app.get('port'), function() {
+	console.log('running on port', app.get('port'));
+});
```
Comment (sudheesh001) [inline slack/index.js]: Break new line here please.

**Comment by shivenmian** [comment]
Okay, fixing this.

**Comment by shivenmian** [comment]
@sudheesh001 @mariobehling To make it better, I used a Slack API wrapper to write this. I'm also not using a webhook anymore. All I do is run the code from my terminal and the bot runs.

The only drawback is that in line 27 of index.js, I had to hardcode my bot id. This was because the wrapper I was using had an error in the method they said would return the bot id (i'd checked their code), so I could not use it.

The bot runs from my terminal as of now. Will deploy onto Heroku later. Please check.

Screenshot:
screen shot 2016-08-05 at 2 02 14 am

**Comment by sudheesh001** [comment]
Needs refactoring as discussed and removal of merge conflicts.

**Comment by shivenmian** [comment]
Yes, will fix.

**Pull request merged**